sup3r/utilities/utilities.py

```diff
@@ -26,6 +26,130 @@
 logger = logging.getLogger(__name__)
 
 
+class DistributedProcess:
```

Comment:
Suggest this goes into `utilities/execution.py` or similar.... the utilities.py file is getting totally ridiculous.

Reply:
Yeah I figured you'd have a suggestion for where to put this. I didn't like it here either.